app/auth/login/index.tsx

```diff
@@ -0,0 +1,111 @@
+import OrgLogin from '@/components/Login/OrgLogin';
+import UserLogin from '@/components/Login/UserLogin';
+import { client } from '@/graphql/client';
+import { LOGIN_MUTATION, ORG_LOGIN_MUTATION } from '@/graphql/mutations/login.mutation';
+import { UserContext } from '@/hook/useAuth';
+import { useMutation } from '@apollo/client';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import { useRouter } from 'expo-router';
+import { useContext, useState } from 'react';
+import { ToastAndroid, Alert } from 'react-native';
+
+export default function SignInOrganization() {
+  const router = useRouter();
+  const [orgLoginSuccess, setOrgLoginSuccess] = useState(false);
+  const [orgLoginMutation] = useMutation(ORG_LOGIN_MUTATION);
+  const [loading, setLoading] = useState(false);
+  const { login } = useContext(UserContext) || {};
+  const [LoginUser] = useMutation(LOGIN_MUTATION);
+
+  const onOrgSubmit = async (values: any) => {
+    console.log('Submitting values:', values);
+    try {
+      await orgLoginMutation({
+        variables: {
+          orgInput: {
+            name: values.organization,
+          },
+        },
+        onCompleted(loginOrg) {
+          console.log('Login successful:', loginOrg);
+          setOrgLoginSuccess(true);
+        },
+        onError(err: any) {
+          ToastAndroid.show(`${err}`, ToastAndroid.LONG);
+        },
+      });
+    } catch (err) {
+      console.log('Mutation error:', err);
+    }
+  };
+
+  const onSubmit = async (userInput: any) => {
+    try {
+      const orgToken = await AsyncStorage.getItem('orgToken');
```

Comment:
Where is this `orgToken` saved? You should save it in `orgLoginMutation` when when an organization is found.

Reply:
### Done.